server/xpub/entities/manuscript/resolvers.js

```diff
@@ -10,6 +14,8 @@ const path = require('path')
 const crypto = require('crypto')
 const Joi = require('joi')
 
+const { ON_UPLOAD_PROGRESS } = asyncIterators
```

Comment:
Is it confusing that this `ON_UPLOAD_PROGRESS` is a string but [the other thing](https://github.com/elifesciences/elife-xpub/pull/368/files#diff-77d316c88c9f296c72d8fd6f9740d4f6R21) called `ON_UPLOAD_PROGRESS` is a graphql operation?

Reply:
Not really, one is client side and the other server side. They had similar examples in apollo as far as I remember.